client/state/selectors/test/get-editor-close-config.js

```diff
@@ -24,7 +24,12 @@ const blockEditorAction = { type: ROUTE_SET, path: '/block-editor/page/1' };
 const checklistAction = { type: ROUTE_SET, path: checklistUrl };
 const customerHomeAction = { type: ROUTE_SET, path: customerHomeUrl };
 
-describe( 'getEditorCloseUrl()', () => {
+// A small wrapper to translate the old tests into the new format.
+function getEditorCloseUrl() {
+	return getEditorCloseConfig( ...arguments ).url;
+}
```

Comment:
Nice shortcut cowboy, but why not actually update the tests instead?

Reply:
because this was easier :(

Reply:
it wasn't really easier, I guess. I think I just wanted there to be less changes to look at

Reply:
updated now :)